src/admin/index.js

```diff
@@ -687,9 +667,8 @@ module.exports = ({
    */
   const listGroups = async () => {
     let groups = []
+    await cluster.refreshMetadata()
```

Comment:
I suspect this was done intentionally once for each node, to avoid having stale metadata. This could just be turned into a `Promise.all` instead of looping through each broker one by one though.

Reply:
I thought so too, but I have a couple issues with it...

1. I dug through `refreshMetadata` and apart from the `shuffle` present in `findConnectedBroker`, I don't see any way to change or control which connected broker is used for the connection. Repeatedly refreshing in this method seems an unreliable, at best, method of grabbing metadata from each broker.
2. Pulling metadata from each broker seems unnecessary. It's the same metadata, right? If it's not the same metadata, then the cluster is in a split brain state and we have bigger problems. This is especially true since the topic metadata isn't used. I believe only the broker list is required to fulfill this particular request.

Let me know your thoughts. I'll move the refresh back into the loop and change it to a `Promise.all` if you still feel it's better to keep it where it is.